charts/kubernetes-view/templates/cluster-overview.yaml

```diff
@@ -70,17 +71,65 @@ spec:
             viewvar__cluster: row.cluster
           name: >
             'namespace'
+    - name: cpu_usage
+      type: gauge
+      unit: millicore
+      description: CPU usage
+      gauge:
+        thresholds:
+          - percent: 0
+            color: '#939b89ff'
```

Comment:
Can we add green < 50, yellow < 75, red thresholds for cpu, and < 70, 85, for memory